.github/workflows/assign-default-project.yml

```diff
@@ -0,0 +1,33 @@
+name: Assign default project to issues
+
+on:
+  issues:
+    types:
+    - opened
+
+jobs:
+  assign_default_project:
+    name: Assign default project to issue
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Assign vulnerability scans or Sentry issues to Enable
+      if: contains(github.event.issue.labels.*.name, 'tool:trivy') || contains(github.event.issue.labels.*.name, 'tool:sentry')
+      uses: m-vdb/create-or-update-project-card@9c632af4932aabf8115780d762841f3ca07f7b6b
+
+      with:
+        token: ${{ secrets.RASABOT_ASSIGN_ISSUES_BOARD_TOKEN }}
+        # https://github.com/orgs/RasaHQ/projects/40
+        project-location: RasaHQ
+        project-number: 40
+        column-name: Inbox
+
+    - name: Notify slack on failure
+      if: failure()
+      env:
+        SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+      uses: voxmedia/github-action-slack-notify-build@v1
```

Comment:
optional: there is also a v2 (but also a newer 1.5 release 🙈 )

Reply:
I pined the latest commit c227368e592